docs/variants/novacustom_ns5x_adl/test-matrix.md

```diff
@@ -38,7 +38,12 @@ subjected before the release of the new binary.
 | 27.  | [QubesOS support][QBS]                                 | QBS           | All                                  |
 | 28.  | [Fedora support][FED]                                  | FED           | All                                  |
 | 29.  | [Platform suspend and resume][SUSP]                    | SUSP          | Without SUSP004.001 and SUSP006.001  |
-| 30.  | [BIOS menu function keys][BMF]                         | BMF           | All                                  |
+| 30.  | [Boot blocking][BBB]                                   | BBB           | All                                  |
+| 31.  | [Reset to defaults][RTD]                               | RTD           | All                                  |
+| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]           | SMS           | All                                 |
```

Comment:
```suggestion
| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]          | SMS           | All                                 |
```

Reply:
Fixed in 4856d86caf7b885bf1645c10375a7d999fd4ab71

Reply:
Had to do that by mistake, fixed in 53b374c64145ec09e899ea4854ae1874ca8a0473